cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -85,13 +68,13 @@ bool FliImpl::isValueLogic(mtiTypeIdT type)
     return false;
 }
 
-bool FliImpl::isValueChar(mtiTypeIdT type)
+bool fli_is_char(mtiTypeIdT type)
 {
     const int NUM_ENUMS_IN_CHAR_TYPE = 256;
     return (mti_TickLength(type) == NUM_ENUMS_IN_CHAR_TYPE);
 }
 
-bool FliImpl::isValueBoolean(mtiTypeIdT type)
+bool fli_is_boolean(mtiTypeIdT type)
```

Comment:
Is this function used anywhere but in this file? If not, these should all be within
```C++
/** local helper functions */
namespace {

}
```
Which is the C++ way of spelling static

Reply:
Agreed.